src/data_provider/tests/sysInfoNetworkBSD/sysInfoNetworkBSD_test.cpp

```diff
@@ -111,7 +110,7 @@ TEST_F(SysInfoNetworkBSDTest, Test_AF_LINK)
     EXPECT_EQ("eth01",ifaddr.at("name").get_ref<const std::string&>());
     EXPECT_EQ("ethernet",ifaddr.at("type").get_ref<const std::string&>());
     EXPECT_EQ("up",ifaddr.at("state").get_ref<const std::string&>());
-    EXPECT_EQ("00:A0:C9:14:C8:29",ifaddr.at("mac").get_ref<const std::string&>());
+    EXPECT_EQ("00:A0:C9:14:C8:29",ifaddr.at("MAC").get_ref<const std::string&>());
```

Comment:
In `networkInterfaceBSD.cpp` file, all values are lower case. This key (and the one changed in line 123) should be updated back to lower case.